smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/httpResponse/bindingTraits/HTTPResponseTraitPayload.kt

```diff
@@ -8,35 +8,38 @@ package software.amazon.smithy.swift.codegen.integration.httpResponse.bindingTra
 import software.amazon.smithy.model.knowledge.HttpBinding
 import software.amazon.smithy.model.shapes.Shape
 import software.amazon.smithy.swift.codegen.SwiftWriter
+import software.amazon.smithy.swift.codegen.integration.HTTPProtocolCustomizable
 import software.amazon.smithy.swift.codegen.integration.HttpBindingDescriptor
 import software.amazon.smithy.swift.codegen.integration.ProtocolGenerator
-import software.amazon.smithy.swift.codegen.integration.httpResponse.HttpResponseBindingRenderable
+import software.amazon.smithy.swift.codegen.integration.httpResponse.HTTPResponseBindingRenderable
 
-interface HttpResponseTraitPayloadFactory {
+interface HTTPResponseTraitPayloadFactory {
```

Comment:
Many types renamed. Also, this type takes customizations as a parameter instead of the error shape.